docs/installing/compiler.md

```diff
@@ -19,68 +19,74 @@ To obtain the newest package:
       },
       "keep": true
     }
   ]
   ```
-  - Create a file .vscode/settings.json with the following content, replacing the extension name/version so that it matches your filesystem. The generator will be chosen automatically, if you use the MinGW toolchain, `mingw32-make` should be on your `PATH` and it will pick the first, for MSVC, you may see a message `[proc] The command: mingw32-make --version failed with error: Error: spawn mingw32-make ENOENT` in the Output tab, but that's ok, MSVC will just use Ninja.
-  ```json5
-  {
-    "cmake.environment": {
-        "BARTMAN_PREFIX_PATH": "${userHome}/.vscode/extensions/bartmanabyss.amiga-debug-1.6.7/bin/win32/"
-    },
-    "cmake.preferredGenerators": ["MinGW Makefiles", "Ninja"]
-  }
-  ```
   - Create empty CMakeLists.txt file **and restart the editor** so that it can discover that you're inside CMake-based project.
-    (You can do this via command palette (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>) and typing "Reload Window".). If your user
-    config already has CMake kits, VSCode may ask you which kit you want to use, then select the Bartman kit from the options.
+    (You can do this via command palette (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>) and typing "Reload Window".).
 
-After this you should be good to go. You can probably check in the `.vscode/cmake-kits.json` file and just adapt the `BARTMAN_PREFIX_PATH` environment variable setting on each machine. You may even want to check in the `.vscode/settings.json` file if you want to make sure that you use the same version of Bartman's extension on all development machines.
+After this you should be good to go. You can check in the `.vscode/cmake-kits.json` file, as long as you keep `AmigaCMakeCrossToolchains`
+in the same relative path to the project. You will be able to compile your project on Windows, macOS, and Linux without any further config.
 
 - From the command palette, select "CMake Configure" to configure your project.
 - The <kbd>F7</kbd> key builds your project,
 - The <kbd>F5</kbd> key runs currently selected debugging task.
 
 Note: The Bartman's compiler produces ELF executables instead of Hunk file format.
 To mitigate this, you need to invoke `elf2hunk` bundled with Bartman's toolchain.
-You may want to look at how its made in CMakeLists of some other projects (e.g. [GermZ](https://github.com/tehKaiN/germz)) to see how ACE is included and debugger is invoked. Also, you may want to further investigate `CMakeLists.txt` there and `.vscode/launch.json` to see how debugger is set up.
+You may want to look at how its made in CMakeLists of some other projects (e.g. [GermZ](https://github.com/tehKaiN/germz)) to see how ACE is
+included and the debugger is invoked there. Also, you may want to further investigate `CMakeLists.txt` there and `.vscode/launch.json` to
+see how debugger is set up, as well as `.vscode/c_cpp_properties.json` to get auto-completion working.
 
```

Comment:
```suggestion

You may also want to set up the CMake kit once on your local machine instead of per-project basis. To do so, follow the steps below:

- from command palette, launch "CMake: Scan for Kits" command to generate the kit configuration,
- then, access your user's CMake kit config by using the "CMake: Edit User-Local CMake Kits" command,
- follow the steps above to tweak the json configuration file.

```